_posts/2021-06-28-Synchronizing-physical-clocks-among-different-UnetStack-based-modems.md

```diff
@@ -132,4 +134,44 @@ phy.clockCalib = mB/mA
 
 ```
 
-Note the parameters used in the above code snippet such as `interval` and `noOfMeasurements` can be varied as per the user's need. The code snippet shown in this blog is to serve as a reference only.
+# Calibrating clock without relying on RTC
+
+It is possible that the RTC is no more accurate than the physical clock. In that case, it is obvious that it should not be used as a reference time source. Instead, we can measure and compare the clocks of the two modems directly. In this section, we describe through a simple python script on how one can achieve this. 
+
+_On user's machine_:
+
+```python
+import time
+from unetpy import *
+
+# open a UnetSocket to modem A
+modemA = UnetSocket("modemA-IP", 1100)
+
+# open a UnetSocket to modem B
+modemB = UnetSocket("modemB-IP", 1100)
+
+# get physical agent ids on modem A and B
+phyA = modemA.agent("phy")
+phyB = modemB.agent("phy")
+
+# measure phy.time on both modem A and modem B
+
+interval = 3600 # 1 hour in seconds
+
+noOfMeasurements = 24
+
+phytimeA = []
+phytimeB = []
+
+for i in range(noOfMeasurements):
+	phytimeA.append(phyA.time)
+	phytimeB.append(phyB.time)
+	time.sleep(interval)
+
+# compute slope based on measurements
+m = (phytimeB[noOfMeasurments-1] - phytimeB[0])/(phytimeA[noOfMeasurments-1] - phytimeA[0]) // slope
+```
+Now based on the computed slope (m), the modem A or modem B's `clockCalib` parameter can be set accordingly to match the other modem's clock as explained in previous section. 
```

Comment:
You probably want to give the exact expression of what value you'd set on which modem.

Reply:
Fixed.